features/admin.branding.v1/constants/ai-branding-constants.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+export const AI_BRANDING_FEATURE_ID: string = "branding.ai1";
```

Comment:
Hope that `1` in the feature id is intentional.

```suggestion
export const AI_BRANDING_FEATURE_ID: string = "branding.ai";
```

Reply:
this is infact has been left after being used for local testing. it is fixed with [address comments](https://github.com/wso2/identity-apps/pull/5961/commits/666a22cf942550d7b1790a0676ff7103f65ebf7a)